buildSrc/src/main/groovy/com/thoughtworks/go/build/WindowsPackagingTask.groovy

```diff
@@ -73,10 +74,18 @@ class WindowsPackagingTask extends DefaultTask {
       downloadAction.execute()
 
       project.copy {
-        from project.tarTree(project.fileTree("${winRootDir()}/jdk-download").include("*").singleFile)
+        from project.tarTree(project.fileTree("${winRootDir()}/jdk-download").singleFile)
```

Comment:
@ketan -- At this point, the file is still a `tar.gz` file, where we can not `exclude` anything.
We need to perform the removal of `src.zip` after the tar has been extracted into buildRoot.

Reply:
it's possible to `project.tartree().matching {exclude '...'}` but what you have works as well.

Can remove the println statement.